tool/tbot/spiffe.go

```diff
@@ -21,22 +21,48 @@ package main
 import (
 	"context"
 	"fmt"
+	"log/slog"
 	"time"
 
 	"github.com/gravitational/trace"
 	"github.com/spiffe/go-spiffe/v2/svid/jwtsvid"
 	"github.com/spiffe/go-spiffe/v2/workloadapi"
-
-	"github.com/gravitational/teleport/lib/utils"
 )
 
+// TODO(tross/noah): Remove once go-spiff has a slog<->workloadapi.Logger adapter.
+// https://github.com/spiffe/go-spiffe/issues/281
+type logger struct {
+	l *slog.Logger
+}
+
+func (l logger) Debugf(format string, args ...interface{}) {
+	//nolint:sloglint // msg cannot be constant
+	l.l.DebugContext(context.Background(), fmt.Sprintf(format, args...))
```

Comment:
Maybe:

```go
	l.log(slog.LevelDebug, format, args...)
```

and implement the l.log function so it centralized the nolint / fmt.Sprintf logic?